ci/github/system-testing/swarm-deploy.bash

```diff
@@ -26,7 +26,6 @@ test() {
   # add too much overhead (e.g. low log-level etc)
   pytest \
     --color=yes \
-    --cov-report=term-missing \
```

Comment:
that is why I think we should get rid of these .bash files. we have 3 different ways of calling pytest in at least 2 different places.
Also I do not really see the problem of calling with term-missing, since this makes the CI output independent of codecov display which I find is a bit misleading sometimes.